src/test/java/bo/usfx/luis/fernando/villca/RowWeightsTests.java

```diff
@@ -0,0 +1,27 @@
+package bo.usfx.luis.fernando.villca;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class RowWeightsTests {
+    @Test
+    public void basicTests() {
+        Assert.assertEquals(new int[]{80, 0}, RowWeights.rowWeights(new int[]{80}));
```

Comment:
Please swap the assertion values:
```java
Assert.assertEquals(RowWeights.rowWeights(new int[]{80}), new int[]{80, 0});
```
Apply this in all your assertions. That is the correct order.